runtime/centrifuge/src/lib.rs

```diff
@@ -605,7 +598,7 @@ impl pallet_vesting::Config for Runtime {
 	type BlockNumberToBalance = ConvertInto;
 	type MinVestedTransfer = MinVestedTransfer;
 	type WeightInfo = pallet_vesting::weights::SubstrateWeight<Self>;
-	const MAX_VESTING_SCHEDULES: u32 = 28;
+	const MAX_VESTING_SCHEDULES: u32 = 3;
```

Comment:
Why 3 only? I guess we will need to change the migration auxiliary code to change the destination format when migrating vesting schedules from one chain to another.

Reply:
3 is from Kusama I think. We are iterating in vesting pallet. Not sure about the number though.

Reply:
We don't even need to change the migration from our off-chain side. Tested this already.